modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -29,9 +34,32 @@ NOTE: Alternatively, you could run `sudo apt install -y redpanda-fips`, which al
 
 If you wish to only use `rpk` on a FIPS host, run:
 
+[,bash]
+----
+sudo yum install -y repdanda-fips redpanda-rpk-fips
+----
+
+--
+Debian/Ubuntu::
++
+--
+To install Redpanda for FIPS compliance, run:
+
+[,bash]
+----
+sudo apt install -y redpanda-rpk-fips redpanda-fips
+----
+
+NOTE: Alternatively, you could run `sudo install -y redpanda-fips`, which also picks up and includes the `redpanda` install package.
```

Comment:
```suggestion
NOTE: Alternatively, you could run `sudo install redpanda-fips`, which also picks up and includes the `redpanda` install package.
```

Reply:
no package manager here?

Reply:
The command should be sudo apt install redpanda-fips for Debian/Ubuntu and sudo yum install redpanda-fips for Rhel